homeassistant/components/image/__init__.py

```diff
@@ -380,3 +409,37 @@ async def handle(
     ) -> web.StreamResponse:
         """Serve image stream."""
         return await async_get_still_stream(request, image_entity)
+
+
+async def async_handle_snapshot_service(
+    image: ImageEntity, service_call: ServiceCall
+) -> None:
```

Comment:
It's not great that this service handler is now fully duplicated between camera and image

Reply:
Let's keep them separated 👍